src/components/structures/TimelinePanel.tsx

```diff
@@ -983,13 +982,15 @@ class TimelinePanel extends React.Component<IProps, IState> {
             ).catch(async (e) => {
                 // /read_markers API is not implemented on this HS, fallback to just RR
                 if (e.errcode === 'M_UNRECOGNIZED' && lastReadEvent) {
-                    const privateField = await getPrivateReadReceiptField(MatrixClientPeg.get());
-                    if (!sendRRs && !privateField) return;
+                    if (
+                        !sendRRs
+                        && !MatrixClientPeg.get().doesServerSupportUnstableFeature("org.matrix.msc2285.stable")
```

Comment:
No issue for this PR . But „does server support **unstable** feature“ … „org.matrix.msc.2285.**stable**“ is confusing.